pkg/actuators/machine/actuator.go

```diff
@@ -423,7 +427,7 @@ func (a *Actuator) Update(context context.Context, machine *machinev1.Machine) e
 		a.handleMachineError(machine, mapierrors.UpdateMachine("no instance found, reason unknown"), updateEventAction)
 
 		// Update status to clear out machine details.
```

Comment:
I totes think we shouldn't clear out the existing details here, but out of scope for this PR.